spec/services/claims/context_mapper_spec.rb

```diff
@@ -78,10 +78,10 @@
       [
         [%w(advocate),                  [:agfs]],
         [%w(litigator),                 [:lgfs]],
-        [%w(admin),                     [:agfs, :lgfs]],
-        [%w(litigator admin),           [:agfs, :lgfs]],
-        [%w(advocate admin),            [:agfs, :lgfs]],
-        [%w(admin advocate litigator),  [:agfs, :lgfs]]
+        [%w(admin),                     %i[agfs lgfs]],
+        [%w(litigator admin),           %i[agfs lgfs]],
+        [%w(advocate admin),            %i[agfs lgfs]],
+        [%w(admin advocate litigator),  %i[agfs lgfs]]
```

Comment:
Just for the sake of consistency you might consider also making this change for the previous two lines too.